udata/settings.py

```diff
@@ -494,6 +494,7 @@ class Testing(object):
         'title^2',
         'description',
     )
+    FS_IMAGES_OPTIMIZE = True
```

Comment:
Somehow this has been overwritten from Defaults while migrating deps in this PR.